<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QMainWindow" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="pixmap">
     <pixmap>img/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Aplikasi penjadwalan sidang skripsi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>By : Muhammad Royyan - 210511081</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>logo</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actDosen"/>
    <addaction name="actMahasiswa"/>
    <addaction name="actFakultas"/>
   </widget>
   <widget class="QMenu" name="menuPenjadwalan">
    <property name="title">
     <string>Penjadwalan</string>
    </property>
    <addaction name="actSidang"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actLogout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actJadwalSidang"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMenu"/>
   <addaction name="menuPenjadwalan"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actDosen">
   <property name="text">
    <string>Dosen</string>
   </property>
  </action>
  <action name="actMahasiswa">
   <property name="text">
    <string>Mahasiswa</string>
   </property>
  </action>
  <action name="actMk">
   <property name="text">
    <string>Mata Kuliah</string>
   </property>
  </action>
  <action name="actSidang">
   <property name="text">
    <string>Sidang</string>
   </property>
  </action>
  <action name="actFakultas">
   <property name="text">
    <string>Prodi</string>
   </property>
  </action>
  <action name="actJadwalSidang">
   <property name="text">
    <string>Jadwal Sidang</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
